content/_partners_impact_stories/dol._en.md

```diff
@@ -0,0 +1,66 @@
+
+
+    As states increased the use of online access to file for UI benefits, the delivery of benefits exposed the vulnerability of online filing systems to identify fraud. States needed tools to secure equitable access to benefits while providing sufficient security to verify identity.
+
+solution: >-
+    ### The solution
+
+    The Department of Labor (DOL) recognized the importance of modernizing state unemployment insurance programs, and the critical role that identity verification played in doing it well. They worked with U.S. Digital Services (USDS) to develop a plan, bringing Login.gov into the fold because of its intuitive user experience and its data privacy principles. The DOL team had an ambitious plan, and Login.gov worked closely with them to quickly launch a pilot program in Arkansas in March 2022.
+
+impact: >-
+    ### The impact
+
+    To date, Login.gov has helped over 20,000 Arkansans more easily access their unemployment benefits.* The pilot was a success, with DOL accelerating the decision to make Login.gov an ongoing offering for Arkansas UI applicants.
+
+
+    The success of the partnership attracted attention from other states who were also working to modernize their unemployment insurance processes, and in June of 2023, DOL announced plans to offer Login.gov as a digital identity option to these interested states. 
+
+
+    Soon after this announcement, Login.gov began planning an integration with the Department of Labor and the State of Hawaii. When the tragic wildfires hit Maui, we accelerated our rollout in order to help folks who would be turning to that service to help them recover. Thousands of Hawaii residents were able to quickly get what they needed in the midst of a crisis, while also having the peace of mind knowing that Login.gov’s security and anti-fraud controls were protecting them.
+
+
+    And because Login.gov is also used for FEMA services via Department of Homeland Security, and disaster assistance loans via Small Business Association, residents of Hawaii could use a single Login.gov account to simplify their interactions with Government services when it mattered most.
+
+
+    State unemployment insurance offices are on the front lines of providing critical services, and the success of this partnership with the Department of Labor highlights the importance of digital identity solutions in supporting this important work. Utah and Massachusetts have joined Arkansas and Hawaii in offering UI identity verification through Login.gov. To date, more than 112,000 digital claimants have been served.**  GSA looks forward to continued work with the Department of Labor to support more states and their residents.
+
+
+    *As of January 2024
+
+
+    **As of March 2024
+
+
+callout_tips: >-
+    * Login.gov has verified over 20,000 unemployment insurance applicants from Arkansas since March 2022
```

Comment:
There's a default margin on lists that creates a noticeable visual offset:

![image](https://github.com/GSA-TTS/identity-site/assets/1779930/4e8a5242-c739-4a58-b2dd-5cc8fcfe518f)

We could either add a utility class to reset the margin, or add a general style to existing `.partners-blue-box` to remove any margin from the first item / first `.usa-list` item?

```suggestion
    {: .margin-top-0}

    * Login.gov has verified over 20,000 unemployment insurance applicants from Arkansas since March 2022
```

Reply:
@aduth We can remove space if you'd like.

While there are some differences between this instance and say the instance at `partners/get-started/`, the other location generally has more "padding" as there's a title.

Adding `.margin-top-0` to follow `usa-list` works.

The text size and padding could be further reviewed by a designer.

Reply:
I see the changes in a46a980, but was there a reason the suggestion in my previous comment wouldn't work as a simpler alternative, and a syntax we use pretty frequently throughout the project?

Reply:
Ah, thank you! I wasn't familiar with that syntax.